ports/re8/port.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 3,
+  "name": "re8.zip",
```

Comment:
why re8?
It's a demake of 8 but the game name is residentvania not resident evil 8.